spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -233,6 +233,19 @@
       )
     end
 
+    it 'redirects unsuccessful authentication for failed reCAPTCHA to failed page' do
+      user = create(:user, :fully_registered)
+
+      allow(FeatureManagement).to receive(:sign_in_recaptcha_enabled?).and_return(true)
+      allow(IdentityConfig.store).to receive(:recaptcha_mock_validator).and_return(true)
+      allow(IdentityConfig.store).to receive(:sign_in_recaptcha_score_threshold).and_return(0.2)
+      stub_analytics
```

Comment:
Do we need to stub analytics if we're not asserting anything about what's being logged?

```suggestion
```